examples/mongodbatlas_project/README.md

```diff
@@ -1,7 +1,56 @@
 # MongoDB Atlas Provider -- Atlas Project with custom limits
-This example creates a Project and defines custom values for certain limits.
+This example creates a Project with tags and defines custom values for certain limits.
+It also shows how you can ignore a specific tag key.
 
 Variables Required to be set:
 - `public_key`: Atlas public key
 - `private_key`: Atlas  private key
 - `org_id`: Organization ID where project will be created
+
+## How to ignore tags managed outside of Terraform
```

Comment:
```suggestion
```

Reply:
As per discussion yesterday, I don't think we should encourage users to manage infra outside of their configs. Looking at ignore_changes [documentation](https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle#ignore_changes), it's not intended so that resources can be managed outside of TF configs, it is intended for other scenarios:

>The ignore_changes feature is intended to be used when a resource is created with references to data that may change in the future, but should not affect said resource after its creation. In some rare cases, settings of a remote object are modified by processes outside of Terraform, which Terraform would then attempt to "fix" on the next run. In order to make Terraform share management responsibilities of a single object with a separate process, the ignore_changes meta-argument specifies resource attributes that Terraform should ignore when planning updates to the associated remote object.


It's really great that we are supporting and being vary of useful TF constructs like `ignore_changes` but at the same time it's important to not miscommunicate the intention here.

Reply:
(sorry about the noise on this) okay so looking at other provider's documentation regarding ignoring tags, I think the intention is conveyed much better [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/resource-tagging#ignoring-changes-to-specific-tags) maybe we can include something along those lines as well

Reply:
Yea. I'm not sure about this either 🤔 

- Pro: It is helpful to know about this method, but it is not intuitive until you see an example.
- Con: It is a bad practice.

I can see two options:
1. Remove the section
2. Add a disclaimer saying this is not recommended and a link to that reference or rewrite it.

I'm curious to hear what other people think, too.

Reply:
I'm not saying that we should say that it isn't recommended but maybe just add more about what scenarios this could be helpful for instead of just "manage it outside of Terraform"

Reply:
I would agree this can be a smaller mention, simply having the usage of tags in the examples/mongodbatlas_project/main.tf file and include a ignore lifecycle block with a comment mentioning that it is possible to ignore specific tag keys.

Reply:
How can we unblock this? should we go with a small mention mostly calling the official documentation and saying that it is possible (but not encouraged)?

Reply:
done in 30aa29163faa90e3fe2c93d467669dd8205f7ec4

Reply:
Removed and moved example to `main.tf`